src/main/java/io/phdata/streamliner/schemadefiner/mapper/CatalogMapper.java

```diff
@@ -0,0 +1,99 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+package io.phdata.streamliner.schemadefiner.mapper;
+
+import io.phdata.streamliner.schemadefiner.model.*;
+import java.util.List;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
+import schemacrawler.crawl.StreamlinerCatalog;
+import schemacrawler.schema.Schema;
+import schemacrawler.schema.Table;
+
+public class CatalogMapper {
+
+  private final Configuration ingestConfig;
+  private final StreamlinerCatalog catalog;
+  private final String jdbcPassword;
+
+  public CatalogMapper(
+      final Configuration ingestConfig,
+      final StreamlinerCatalog catalog,
+      final String jdbcPassword) {
+    this.ingestConfig = ingestConfig;
+    this.catalog = catalog;
+    this.jdbcPassword = jdbcPassword;
+  }
+
+  public Configuration mapJdbcCatalogToConfig() {
+    Jdbc jdbc = (Jdbc) ingestConfig.getSource();
```

Comment:
I think this isn't really part of your PR, but what else can a source be that you would have to cast it here?

Reply:
I Agree. This was not needed for this PR. But I saw most of the code was using static methods and constant fields were passed to methods to access it. 
I changed them to non-static and made constant fields as class-level parameters so that any method can access them. 
This simplifies getting the JDBC connection for Hive.

We have two sources GlueCatalog and JDBC.  In this method, it will be JDBC.